src/tests/Common/XUnitLogChecker/XUnitLogChecker.csproj

```diff
@@ -5,15 +5,18 @@
     <TargetFramework>$(NetCoreAppToolCurrent)</TargetFramework>
     <GenerateRuntimeConfigurationFiles>true</GenerateRuntimeConfigurationFiles>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+    <PublishAot>true</PublishAot>
+    <SuppressTrimAnalysisWarnings>false</SuppressTrimAnalysisWarnings>
+    <LinkerFlavor Condition="'$(CrossBuild)' == 'true' and '$(TargetsLinux)' == 'true'">lld</LinkerFlavor>
```

Comment:
For my own education, how does this affect the output?

Reply:
`PublishAot` is what says, when we run publish, AOT compile.
`SuppressTrimAnalysisWarnings=false` enables warnings about AOT incompatibility.
`LinkerFlavor=lld` sets us to use the `lld` linker, which is the one we use for all of the runtime